.github/workflows/main-bulids.yml

```diff
@@ -0,0 +1,165 @@
+name: Build transformer image for production env
+on:
+  push:
+    branches:
+      - 'main'
+env:
+  REPO_NAME_OLD: rudderlabs/rudder-transformer
+  REPO_NAME: rudderstack/rudder-transformer
+  DOCKERHUB_USERNAME: rudderlabs
+
+jobs:
+  transformer-build:
+    name: Build transformer image
+    environment: production
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get_version.outputs.version }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get image version
+        id: get_version
+        run: |
+          VERSION=$(jq -r .version package.json)
+          DATEVERSION="$(date '+%d%m%Y.%H%M%S')"
+          echo $VERSION
+          echo $DATEVERSION
+          echo "::set-output name=version::$VERSION"
+          echo "version=$VERSION" >> "$GITHUB_OUTPUT"
+          echo "date-version=$DATEVERSION" >> "$GITHUB_OUTPUT"
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ env.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PROD_TOKEN }}
+
+      - name: Build image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: Dockerfile
+          load: true
+          tags: ${{ env.REPO_NAME }}:${{ steps.get_version.outputs.version }}
+
+      - name: Run Tests
+        run: |
+          docker run ${{ env.REPO_NAME }}:${{ steps.get_version.outputs.version }} npm run test
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and push multi-platform images
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: Dockerfile
+          push: true
+          tags: |
+            ${{ env.REPO_NAME_OLD }}:${{ steps.get_version.outputs.version }}
+            ${{ env.REPO_NAME }}:${{ steps.get_version.outputs.version }}
+            ${{ env.REPO_NAME }}:latest
+            ${{ env.REPO_NAME }}:${{ steps.get_version.outputs.date-version }}
```

Comment:
Actually, let's get rid of the date version completely. It can lead to ambiguity.

Reply:
removed date version